pkg/kapp/crdupgradesafety/validator.go

```diff
@@ -73,3 +74,20 @@ var NoScopeChangeValidateFunc ValidateFunc = func(old, new v1.CustomResourceDefi
 	}
 	return nil
 }
+
+var NoStoredVersionRemovedValidateFunc ValidateFunc = func(old, new v1.CustomResourceDefinition) error {
```

Comment:
Wondering why this is a `var` rather than just a `func`? The function signature should be enough to indicate that it's a `ValidateFunc`?

Reply:
Fair point. I was doing this so it is more explicit if the `ValidateFunc` type changes, but I do agree that this is too verbose. Updated in https://github.com/everettraven/kapp/pull/4/commits/26c6daa66577cdda6a17fbaafb242a557762d914